tests/PHPStan/Rules/Pure/PureMethodRuleTest.php

```diff
@@ -194,10 +194,12 @@ public function dataBug11207(): array
 		];
 	}
 
-	public function testBug12048(): void
+	public function testBug12224(): void
 	{
 		$this->treatPhpDocTypesAsCertain = true;
-		$this->analyse([__DIR__ . '/data/bug-12048.php'], []);
+		$this->analyse([__DIR__ . '/data/bug-12224.php'], [
```

Comment:
I don't get it, why did you delete this test? You should only add a new test.

Reply:
not sure why i did that. Reverted the delete